app/meteor-accounts-saml/server/saml_server.js

```diff
@@ -10,6 +10,7 @@ import _ from 'underscore';
 import { SAML } from './saml_utils';
 import { CredentialTokens } from '../../models';
 import { generateUsernameSuggestion } from '../../lib';
+import {_setUsername} from '../../lib/server/functions';
```

Comment:
Please always check jslint by doing: `meteor npm run lint`

```suggestion
import { _setUsername } from '../../lib/server/functions';
```

Reply:
Sorry, have updated it!